pkg/driver/camera/camera_linux.go

```diff
@@ -184,6 +185,16 @@ func (c *camera) VideoRecord(p prop.Media) (video.Reader, error) {
 		if err != nil {
 			return nil, err
 		}
+
+		framerate, err := c.cam.GetFramerate()
+		if err != nil {
+			return nil, err
+		}
+
+		if framerate != p.FrameRate {
```

Comment:
Framerate in float may have quantization error.
For example, there are 29.97 fps which can't be represented using IEEE754 float.
(This is why the internal v4l API uses fraction)

I'm not very sure we should return error if actual framerate is not equal to the desired value.
Cameras I have seems to select nearest available framerate if given value doesn't match any of available values.
In some case, user just want nearest framerate instead of the exact one. (on many USB cameras, framerate depends on the brightness of the image)

Reply:
I can take this check out. I think it would be good to at least print something to the console to warn the user if the specified framerate is different that the one reported by the device.

Reply:
Removed in 782d02e00c401f42e8cc0c3ca256ced1f5943afd.